WooCommerce/Classes/ViewRelated/FilterTabBar/TabbedViewController.swift

```diff
@@ -102,8 +102,23 @@ class TabbedViewController: UIViewController {
             return
         }
         tabBarStackView.removeArrangedSubview(customTabBarView)
+        customTabBarView.removeFromSuperview()
         self.customTabBarView = nil
     }
+
+    func appendToTabBar(_ tab: TabbedItem) {
+        // Setup child view controller
+        items.append(tab)
+        tabBar.items = items
+        addChild(tab.viewController)
+        stackView.addArrangedSubview(tab.viewController.view)
+        tab.viewController.didMove(toParent: self)
```

Comment:
❓  Should we set the `tab.viewController.view` as hidden following `configureChildViewControllers` implementation before we update visibility in `updateVisibleChildViewController`?

Reply:
At first I thought we can save some computation by not hiding the custom tab, as it's going to become the selected tab anyway.

But now that I think of it, it might cause some unexpected bug if at some point there's multiple non-hidden view controllers. Added the initial hiding in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/046ac0b3df333cbbf1a19ae46a875408424498e3